### PR TITLE
Features/search bkend 84

### PR DESCRIPTION
**Pull request by dan-crayne**
Sorry for the confusing commit history; I had made some rookies mistakes when merging while I was still in process.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class ApplicationController < ActionController::API
   # https://github.com/gonzalo-bulnes/simple_token_authentication#make-models-token-authenticatable
-  acts_as_token_authentication_handler_for User, except: [:search, :show, :linkedin, :random], fallback: :exception
+  acts_as_token_authentication_handler_for User, except: [:search, :index, :show, :linkedin], fallback: :exception
```
Comment (evwhite29) [inline app/controllers/application_controller.rb]: I don't agree with this change. The whitelist of publicly available endpoints (actions) was cemented in a previous branch and should not be altered.

Comment (dan-crayne) [reply app/controllers/application_controller.rb]: I didn't intend to make that change. I think this is a relic of my code being out of sync with the main branch.

**Review by evwhite29 — DISMISSED** [review]
Looks good!

**Review by iandusenbury — APPROVED** [review]
Looks good to me

**Review by evwhite29 — APPROVED** [review]

**Pull request merged**